README.md

```diff
@@ -51,18 +51,18 @@ The ZMT service assumes that containers are running on the same host as the brow
 
 ### `irods-client-nfsrods` / NFSRODS
 
-Once the service is running, the NFS server needs to be accessed from a mountpoint on the host machine. This can be done with the following command:
+Once the service is running, the NFS server needs to be accessed from a mountpoint. This can be done with the following command:
 ```bash
-$ sudo mount -o sec=sys,port=2049 localhost:/ ./irods_client_nfsrods/nfs_mount
+$ sudo mount -o sec=sys,port=2050 localhost:/ ./irods_client_nfsrods/nfs_mount
 ```
-The hostname can also be the IP address of the container providing the service in the `irods_demo_default` Docker network. For more information about mounting NFSRODS, see the README for the project: https://github.com/irods/irods_client_nfsrods#mounting
+The hostname can also be the IP address of the container providing the service in the `irods_demo_default` Docker network if running from the same host. The mountpoint can exist on any host which can reach the host running the container providing the service because the port is being exposed, in which case the FQDN or IP address for the host machine can be used. For more information about mounting NFSRODS, see the README for the project: https://github.com/irods/irods_client_nfsrods#mounting
```

Comment:
```suggestion
The hostname can also be the IP address of the container providing the service in the `irods_demo_default` Docker network if running from the same host. The mountpoint can exist on any other machine which can reach the host running the container providing the service because the port is being exposed, in which case the FQDN or IP address for the host machine can be used. For more information about mounting NFSRODS, see the README for the project: https://github.com/irods/irods_client_nfsrods#mounting
```